data/types.go

```diff
@@ -0,0 +1,76 @@
+package data
+
+import (
+  "math/big"
+
+  "github.com/harmony-one/harmony/numeric"
+  "github.com/harmony-one/harmony/staking/types"
+)
+
+type NodeMetadataReply struct {
+  BLSKeys      []string `json:"blskey"`
+	Version      string   `json:"version"`
+	NetworkType  string   `json:"network"`
+	ShardID      float64  `json:"shard-id"`
+	NodeRole     string   `json:"role"`
+	ArchivalNode bool     `json:"is-archival"`
+}
+
+type LatestHeaderReply struct {
+	BlockHash        string  `json:"blockHash"`
+	BlockNumber      float64 `json:"blockNumber"`
+	ShardID          uint32  `json:"shardID"`
+	Leader           string  `json:"leader"`
+	ViewID           uint64  `json:"viewID"`
+	Epoch            int     `json:"epoch"`
+	Timestamp        string  `json:"timestamp"`
+	UnixTime         int64   `json:"unixtime"`
+	LastCommitSig    string  `json:"lastCommitSig"`
+	LastCommitBitmap string  `json:"lastCommitBitmap"`
+}
+
+type BlockByNumberReply struct {
+  BlockHash              string   `json:"hash"`
+  BlockNumber            string   `json:"number"`
+  BlockSize              string   `json:"size"`
+  BlockSizeInt           int
+  ParentHash             string   `json:"parentHash"`
+  StateRoot              string   `json:"stateRoot"`
+  Transactions           []string `json:"transactions"`
+  NumTransactions        int
+  StakingTransactions    []string `json:"stakingTransactions"`
+  NumStakingTransactions int
+}
+
+// HACK: To get UnmarshalJSON to not error when unmarshalling one addresses
+type ValidatorInformationReply struct {
```

Comment:
Temp hacks fine but later let’s impl proper unmarshal in main repo as part of bigger issue of rpc refactor